examples/wallet_smart_contract/src/main.sw

```diff
@@ -33,12 +30,12 @@ impl Wallet for Contract {
         }
     }
 
-    #[storage(read, write)]
+    #[storage(read, write), payable]
```

Comment:
Actually how come `send_funds` needs `payable` here? I think only `receive_funds` requires `payable` because it uses `msg_amount` therefore it's expecting assets to be forwarded with the call.

Reply:
Ahh, my bad. I think I understood the use of #payable incorrectly. Thanks for the clarification.